diff --git a/node_modules/@apollo/server/dist/esm/runHttpQuery.js b/node_modules/@apollo/server/dist/esm/runHttpQuery.js
index 96ef0ab..0d341fa 100644
--- a/node_modules/@apollo/server/dist/esm/runHttpQuery.js
+++ b/node_modules/@apollo/server/dist/esm/runHttpQuery.js
@@ -187,6 +187,7 @@ function orderExecutionResultFields(result) {
 }
 function orderInitialIncrementalExecutionResultFields(result) {
     return {
+        ...result,
         hasNext: result.hasNext,
         errors: result.errors,
         data: result.data,
@@ -196,6 +197,7 @@ function orderInitialIncrementalExecutionResultFields(result) {
 }
 function orderSubsequentIncrementalExecutionResultFields(result) {
     return {
+        ...result,
         hasNext: result.hasNext,
         incremental: orderIncrementalResultFields(result.incremental),
         extensions: result.extensions,
@@ -203,6 +205,7 @@ function orderSubsequentIncrementalExecutionResultFields(result) {
 }
 function orderIncrementalResultFields(incremental) {
     return incremental?.map((i) => ({
+        ...i,
         hasNext: i.hasNext,
         errors: i.errors,
         path: i.path,
